### ISSUE TITLE
Change in Matplotlib 3.0.0 breaks the combination of Plots, pyplot(), and LaTeX ticks in scientific notation

### ISSUE THREAD

**Issue by benide** [JuliaPlots/Plots.jl#2194]
As of Matplotlib 3.0.0 (see [here](https://matplotlib.org/devdocs/api/prev_api_changes/api_changes_3.0.0.html)), Matplotlib switched LaTeX rendering from using `\usepackage{ucs}\usepackage[utf8x]{inputenc}`, to `\usepackage[utf8]{inputenc}`. This does not support the unicode used in tick labels coming from `Showoff.jl`.

To reproduce, make sure `matplotlicrc` has the line `text.usetex : True` and in Julia, run 
```julia
using Plots; pyplot()
plot(1:10, 1e6*collect(1:10))
```
#2147 is requesting a change that is one way to fix this problem. I think this is also related to #2042.

**Comment by benide** [comment]
I've hacked up a solution that works for me. In `src/axes.jl` in the function `optimal_ticks_and_labels`, I added a formatter `:latex` that does the following:
```julia
map(x -> string("\$", replace(convert_sci_unicode(x), '×' => "\\times"), "\$"), Showoff.showoff(unscaled_ticks, :auto))
```
This works fine for me. Is this worth a PR? I think it would help both @isentropic and @sswatson with #2147 and #2042, but I'm not sure if this is the kind of thing you would want to add.

**Comment by benide** [comment]
Solved by #2195. Thanks for the quick merge!
